src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,67 +1,97 @@
 import * as React from "react";
 import { Meta, Story } from "@storybook/react/types-6-0";
+import range from "lodash/range";
 import Accordion, { AccordionProps } from "./Accordion";
 import { action } from "@storybook/addon-actions";
+import { withDesign } from "storybook-addon-designs";
 
-import Checkbox from "../Checkbox/Checkbox";
-import List from "../List/List";
-import { ListTypes } from "../List/ListTypes";
+// import List from "../List/List";
+// import { ListTypes } from "../List/ListTypes";
+import { list as ListStory } from "../List/List.stories";
+import Link from "../Link/Link";
 
 export default {
     title: "Accordion",
     component: Accordion,
+    decorators: [withDesign],
 } as Meta;
 
 // Set up the reusable template to create multiple stories for the
 // Accordian component.
-const Template: Story<AccordionProps> = (args) => (
-    <Accordion {...args}>{childrenList}</Accordion>
-);
-
-const checkboxes = [
-    <li key="one">
-        <Checkbox
-            onChange={action("changed")}
-            checkboxId="checkbox-1"
-            labelOptions={{
-                id: "checkbox-1-label",
-                labelContent: <>Checkbox 1</>,
-            }}
-        ></Checkbox>
-    </li>,
-    <li key="two">
-        <Checkbox
-            onChange={action("changed")}
-            checkboxId="checkbox-2"
-            labelOptions={{
-                id: "checkbox-2-label",
-                labelContent: <>Checkbox 2</>,
-            }}
-        ></Checkbox>
-    </li>,
-    <li key="three">
-        <Checkbox
-            onChange={action("changed")}
-            checkboxId="checkbox-3"
-            labelOptions={{
-                id: "checkbox-3-label",
-                labelContent: <>Checkbox 3</>,
-            }}
-        ></Checkbox>
-    </li>,
-];
-const childrenList = (
-    <List type={ListTypes.Unordered} id="checkbox-list">
-        {checkboxes}
-    </List>
+const AccordionTemplate = ({ count, children, ...args }) => (
```

Comment:
I'd call this something else, like `AccordionListTemplate` since it's a list of `Accordion` components and not the component itself. Maybe the returned HTML should be ul/li of `Accordion`s.